src/style.css

```
@@ -1,3 +1,74 @@
+:root {
+  --link-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content:space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+nav ul{
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+nav ul li {
+  margin: 0 10px;
+  position: relative;
+}
+
+ul a {
+  display: flex;
+  text-align: center;
+  line-height: 60px;
+  height: 60px;
+}
+
+nav ul li:first-child {
+  margin-left: 0;
+}
+
+nav ul li:last-child {
+  margin-right: 0;
+}
+
+
+nav a {
+  text-decoration: none;
+  color: #000;
```

if you make a variable for some color - make variables for all colors)
```suggestion
  color: var;
```